### ISSUE TITLE
[BUG] TypeError: Cannot read property 'find' of undefined on @loadable/server 5.14.2

### ISSUE THREAD

**Issue by olso** [gregberge/loadable-components#697]
## 🐛 Bug Report

I use SSR

```bash
TypeError: Cannot read property 'find' of undefined
    at ChunkExtractor.getChunkInfo (/Users/olso/Dev/seasy/web/node_modules/@loadable/server/lib/ChunkExtractor.js:226:41)
    at /Users/olso/Dev/seasy/web/node_modules/@loadable/server/lib/ChunkExtractor.js:291:32
    at Array.filter (<anonymous>)
    at one (/Users/olso/Dev/seasy/web/node_modules/@loadable/server/lib/ChunkExtractor.js:290:32)
    at arrayMap (/Users/olso/Dev/seasy/web/node_modules/lodash/_arrayMap.js:16:21)
    at map (/Users/olso/Dev/seasy/web/node_modules/lodash/map.js:50:10)
    at flatMap (/Users/olso/Dev/seasy/web/node_modules/lodash/flatMap.js:26:22)
    at ChunkExtractor.getChunkDependencies (/Users/olso/Dev/seasy/web/node_modules/@loadable/server/lib/ChunkExtractor.js:302:54)
    at ChunkExtractor.getRequiredChunksScriptContent (/Users/olso/Dev/seasy/web/node_modules/@loadable/server/lib/ChunkExtractor.js:309:32)
    at ChunkExtractor.getRequiredChunksScriptElements (/Users/olso/Dev/seasy/web/node_modules/@loadable/server/lib/ChunkExtractor.js:335:22)
```

## To Reproduce

```js
new LoadablePlugin({
      writeToDisk: { filename: "dist" },
    }),
```

```js
"@loadable/babel-plugin",
```

```js
const extractor = new ChunkExtractor({
  statsFile: path.resolve(__dirname, "../../../dist/loadable-stats.json"),
});
```

## Link to repl or repo (highly encouraged)

Please provide a minimal repository on GitHub.

Issues without a reproduction link are likely to stall.

## Run `npx envinfo --system --binaries --npmPackages @loadable/component,@loadable/server,@loadable/webpack-plugin,@loadable/babel-plugin --markdown --clipboard`

Paste the results here:

I use `yarn` so `yarn list --depth=1 | grep loadable`

```bash
├─ @loadable/babel-plugin@5.13.2
├─ @loadable/component@5.14.1
├─ @loadable/server@5.14.2
├─ @loadable/webpack-plugin@5.14.0
├─ @types/loadable__component@5.13.1
├─ @types/loadable__server@5.12.3

```

**Comment by theKashey** [comment]
This more bug than a feature, but `loadable` is expected(and tested) to work only when all pieces having matching versions.
In your case `plugin` is a little behind `server` and is not providing expected information to it.

----

Your issue can be closed now, as there is a simple fix, but this issue should stay open, as we should enforce version match from our side and this technical moment should not be your problem.

**Comment by peterjcaulfield** [comment]
@theKashey I think the issue here is due to the fact that latest release for package doesn't appear to be published to NPM?

![Screenshot 2021-02-24 at 18 04 19](https://user-images.githubusercontent.com/1675867/109044852-cecdd780-76ca-11eb-8b8a-cd47f443bb84.png)

Based on the tags in this repo this should point to `15.4.2`?

**Comment by wolfflow** [comment]
@theKashey seems that there were no new `plugin` versions released after 5.13.2.
